docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -1,2 +1,101 @@
++
+For more information about the case file, see <<case-management-case-file-con-{context}>>.
+
+. *Define the roles involved in the case.*
+.. Click image:3898.png[] to open the *Properties* menu on the right side of the designer and open the *Editor for Case Roles*. 
+.. Click *Add Case Role* to add case roles.  
++
+The IT Orders sample case management project includes the following roles:
++
+
+.ITOrders Case Roles
+image::case_roles.png[Case Roles]
+
++
+* `owner` - the employee who is making the hardware order request. There can be only one of these roles.
+* `manager` - the employee's manager; the person who will approve or deny the requested hardware. There is only one manager in the IT Orders project.
+* `supplier` - the available suppliers of IT hardware in the system. There is usually more than one supplier.
++
+For more information about creating case roles, see <<case-management-roles-con-{context}>>.
+
+. *Define what work item will be triggered when a new case starts.*
++
+As case definition is an ad hoc process definition, it does not require an explicit start node. Nodes that have no incoming connections and are marked as `Adhoc Autostart` and are automatically triggered when an instance is started.
++
+By default, the case definition designer includes a `Milestone` object configured as `Adhoc autostart` in the palette. Modify this object or replace it with another work item, such as a user task or script task to be triggered when a new case is started.
++
+In the IT Orders example, the `Prepare hardware spec` user task and the `Milestone 1: Order placed` milestones are `Adhoc autostart`, and are automatically triggered when an instance is started by a request being made for new IT hardware.
+
+. *Define the `Place order` subprocess.*
++
+In the IT Orders example, the `place-order` process is a separate business process that is carried out by the supplier. This is a reusable process that occurs during the course of case execution. 
+.. From the project menu, click *Add Asset* -> *Business Process* to create a new business process. Define the name and package for the new process and click *Ok*. 
+.. Define the process that is a subprocess within the case. For example, the IT Orders `Place order` process is a simple user task that applies to the supplier group and receives the relevant order data and case variables directly from the case file.
+
+. *Define milestones*
++
+Add a milestone to the case definition by selecting `Milestone` from the *Object Library* and dragging it on to the case designer. 
+.. Create an evaluation condition for the milestone based on a case file variable that determines when the milestone is activated. From the *Properties* menu on the right, open the *Data Inputs and Assignments* window and click *Add* to add a new condition.
```

Comment:
I'd replace activated with achieved or completed as once the condition is met the milestone is not activated as it must be active to actually receive the event of condition being met.

Reply:
Changed to completed.